### ISSUE TITLE
Multiple MMKV Instances

### ISSUE THREAD

**Issue by georgeflug** [pnthach95/rn-mmkv-storage-flipper#1]
Thank you for the plugin!

It would be nice if this supported multiple MMKV instances. Right now data only shows up for 1 instance and it seems random which one it picks.

e.g.

```
const appPrefs = new MMKVLoader()
  .withInstanceID('appPrefs')
  .initialize();
const userData = new MMKVLoader()
  .withInstanceID('userData')
  .withEncryption()
  .initialize();

if (__DEV__) {
  // both should show up in flipper
  mmkvFlipper(prefs);
  mmkvFlipper(encryptedData);
}
```

**Comment by pnthach95** [comment]
Hi @georgeflug, I just published v1.1.0 to support multiple MMKV instances. Please update your code to this and test:

```js
if (__DEV__) {
  mmkvFlipper([prefs, encryptedData]);
  // Put array of instances ^
}
```

If it's okay, you can close issue.

**Comment by georgeflug** [comment]
Thank you very much!
